MCprep_addon/mcprep_ui.py

```diff
@@ -189,6 +213,13 @@ def change_verbose(self, context):
 			"meshswapable objects and groups"),
 		subtype = 'FILE_PATH',
 		default = scriptdir + "/MCprep_resources/mcprep_meshSwap.blend")
+
+	model_path = bpy.props.StringProperty(
+		name = "Model path",
+		description = ("Default path to the model asset file, for "
+			"models"),
+		subtype = 'FILE_PATH',
+		default = scriptdir + "/MCprep_resources/mcprep_models.blend")
```

Comment:
fyi to generally never explicitly us "/" in path names, as they are OS specific. This would be properly written as: os.path.join(scriptdir, "MCprep_resources", "mcprep_models.blend)